subset/bacnet/uniqidtest/Dockerfile.test_uniqid

```diff
@@ -0,0 +1,22 @@
+FROM daq/aardvark:latest as bacnet_build
+
+# Do this alone first so it can be re-used by other build files.
+RUN $AG update && $AG install openjdk-8-jre
+
+RUN $AG update && $AG install openjdk-8-jdk git
+
+#This refer to a specific github commit on the repo
+ENV BACHASH=6066ce5e0494d778f8efa0bb5ae3bf192b888abd
```

Comment:
We should make sure that this value is in sync with all the other uses of BACHASH in the setup... 

E.g.:

~/daq$ fgrep BACHASH= -r docker/ subset/
docker/Dockerfile.test_bacnet:ENV BACHASH=d97c917a5ad24e6c9b45f49a5b01efffda9e820e
docker/Dockerfile.faux:ENV BACHASH=d97c917a5ad24e6c9b45f49a5b01efffda9e820e